backtracking/rat_in_maze.py

```
@@ -63,16 +63,16 @@ def solve_maze(maze: list) -> bool:
     if solved:
         print("\n".join(str(row) for row in solutions))
     else:
-        print("Solution does not exists!")
+        print("Solution does not exist!")
```

```suggestion
        print("No solution exists!")
```

---

I would raise an exception but that is a personal choice.